client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -1683,9 +1683,13 @@ public void testShrinkIndex() throws Exception {
         request.setWaitForActiveShards(2); // <1>
         request.setWaitForActiveShards(ActiveShardCount.DEFAULT); // <2>
         // end::shrink-index-request-waitForActiveShards
+        // tag::shrink-index-request-copySettings
+        request.setCopySettings(Boolean.TRUE); // <1>
```

Comment:
`true` is more natural

Reply:
Addressed in de8ca1863c29ea222fe23b3366d207a4bdbcb4de